public/app/screens/main/sections/device-details/style.scss

```diff
@@ -46,46 +49,33 @@ $column-factor: $total-columns / $content-columns;
   &__controls,
   &__properties,
   &__access-rights,
-  &__details{
+  &__details {
     @include grid-row();
```

Comment:
So we left grid-row as a mixin. Which classes will be used explicitly? And which functionality we will use as a mixins?